src/logic/hooks/useCanTxExecute.tsx

```diff
@@ -1,29 +1,10 @@
 import { useSelector } from 'react-redux'
-import { extractSafeAddress } from 'src/routes/routes'
 import { currentSafe } from '../safe/store/selectors'
-import useGetRecommendedNonce from './useGetRecommendedNonce'
 
-export const calculateCanTxExecute = (
-  currentSafeNonce: number,
-  preApprovingOwner: string,
-  threshold: number,
-  txConfirmations: number,
-  recommendedNonce?: number,
-  isExecution?: boolean, // when executing from the TxList
-  manualSafeNonce?: number,
-): boolean => {
-  if (isExecution) return true
-
-  // Single owner
-  if (threshold === 1) {
```

Comment:
This is an edge case for our app, and isn't worth all the edge case bugs it introduces.

Reply:
This will hide the option to not execute if it's a single owner. I understand it is not the most common case in a multisig but are we ok with it?

Reply:
I'll make sure to make it clear in the description so that QA is aware.

Reply:
We could also include it at the end to not take priority over the other cases. I think it should be possible to immediately execute a Tx in a single owner / single policy safe.